easybuild/framework/extensioneasyblock.py

```diff
@@ -105,11 +119,11 @@ def run(self, unpack_src=False):
             targetdir = os.path.join(self.master.builddir, remove_unwanted_chars(self.name))
             self.ext_dir = extract_file(self.src, targetdir, extra_options=self.unpack_options,
                                         change_into_dir=False)
-            change_dir(self.ext_dir)
 
-            if self.start_dir and os.path.isdir(self.start_dir):
-                self.log.debug("Using start_dir: %s", self.start_dir)
-                change_dir(self.start_dir)
+            self._set_start_dir()
+            change_dir(self.start_dir)
```

Comment:
Turns out this is a breaking change, because the `isdir(self.start_dir)` check is now being done from a different working directory.

That matters (a lot), because the `self.start_dir` value is usually a path relative to the unpacked sources for the extension.

I'll look into fixing this, and covering it in the tests, this was way too easy to overlook...

Reply:
Good catch! Suggestion (pseudo code): 
```
start_dir = self.start_dir or '.'
ext_dir = self.ext_dir or os.pwd()
os.path.make_absolute(start_dir, ext_dir)
then check for is_dir and use
```

Reply:
problem fixed in #3435